src/Params/DevPingParams.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Avtocod\B2BApi\Params;
+
+final class DevPingParams
```

Comment:
Maybe transfer the prefix `Dev` to the namespace?

Like:
```php
namespace Avtocod\B2BApi\Params\Dev;

final class PingParams{
...
```

Or

```php
namespace Avtocod\B2BApi\Params\User;

final class BalanceParams{
...

Reply:
Looks great, but I doubt the name of the parameter `UserParams`.

This implementation -- is well?

```php
namespace Avtocod\B2BApi\Params\User;

final class InfoParams{
...
```

Reply:
`InfoParams` Looks pretty

Reply:
Now, params-objects with prefixes `Dev` & `User` e35274fefc31dd0d6895b14057a58f5e1d3cda62

Reply:
Great!